peachjam/js/components/DocumentContent/index.ts

```diff
@@ -30,18 +30,25 @@ class DocumentContent {
   private searchApp: any;
   private navOffCanvas: OffCanvas | undefined;
   private enchrichmentsManager: EnrichmentsManager | null;
+  private storedDocument: Node | undefined;
+  private tocController: HTMLElement| null;
   constructor (root: HTMLElement) {
     this.root = root;
     this.navOffCanvas = undefined;
     this.enchrichmentsManager = null;
+    this.storedDocument = this.root.querySelector('[data-document-element]')?.cloneNode(true);
```

Comment:
You run this `querySelector` three times. Better to do it here in the constructor once and store the element for re-use.

Reply:
done